content/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -0,0 +1,295 @@
+website](https://prow.ci.openshift.org/plugins), please consult that list while reading the following section for more
+detail.
+
+Plugin configuration is stored in
+[`_plugins.yaml`](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_plugins.yaml) in the
+[openshift/release](https://github.com/openshift/release) repository. Plugins are enabled for a repository or
+organization under the `plugins` key. Plugin-specific configuration is under keys like `label` or `owners`. The set of a
+repository’s enabled plugins is the union of plugins configured for the repository’s organization (found at the
+`plugins.yaml["plugins"]["$org"]` key) and the repository itself (found at the `plugins.yaml["plugins"]["$org/$repo"]`
+key).
+
+Most individual plugins can be configured to change their behavior; only some plugins allow for granular configuration
+at a repository level, many only expose global configuration options for all repositories that Prow monitors. If you
+think you need to configure an individual plugin, consult a CI administrator. While we work on a better solution,
+documentation for these options lives in the `type Configuration struct`
+[here.](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/plugins.go)
+
+#### Repositories Under Existing Organizations
+
+If you are onboarding a repository in an organization for which plugins are already configured, you will only need to
+enable plugins that you do not already inherit from the organization by adding those plugins under a new
+`plugins.yaml["plugins"]["$org/$repo"]` key. The one plugin you may want to configure at a repository-scoped level is:
+
+|Name|Description|
+|:---|:---|
+|`approve`|enables the `/approve` functionality with `OWNERS` files|
+
+Review the list of plugins enabled for your owning organization and the live [plugin
+catalog](https://prow.ci.openshift.org/plugins) when choosing which plugins you want on your repo.
+
+If your repository does not have `OWNERS files`, you will not be able to opt into the `/approve` process or automatic pull
+request review assignment. `OWNERS` file format and interaction details can be found
+[upstream.](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/approve/approvers/README.md) You will also
+need to add bugzilla component information to your `OWNERS` file. Format can be found
+[here.](https://docs.google.com/document/d/1V_DGuVqbo6CUro0RC86THQWZPrQMwvtDr0YQ0A75QbQ/edit?usp=sharing)
+
+#### Repositories Under New Organizations
+
+If you are onboarding a component not in any organization we already have configured, consider copying the openshift
+organization’s plugin configuration for your organization under a new `plugins.yaml["plugins"]["$org"]` key, or/ setting
+your repo’s configuration without adding any organization-wide configuration by adding the full list of plugins you
+require under a new `plugins.yaml["plugins"]["$org/$repo"]` key.
+
+### Describing Tests
```

Comment:
Can we remove the link to `ci-operator/jobs` and link instead to the `ci-operator` arch overview sections on these types?

Reply:
This section describes the directory structure so replacing it with a link to ci-operator details doesn't really seem appropriate?